### PR TITLE
Update specs for cocoapods-downloader changes

### PR DESCRIPTION
**Pull request by amorde**
Changes from https://github.com/CocoaPods/cocoapods-downloader/pull/98

### CODE REVIEW

File: install_add_pod/after/execution_output.txt [hunk 1]
```diff
@@ -30,16 +30,22 @@ Downloading dependencies
  > Git download
      $ GIT_BIN clone https://github.com/tonymillion/Reachability.git PROJECT/Pods/Reachability --template= --single-branch --depth 1 --branch v3.1.0
      Cloning into 'PROJECT/Pods/Reachability'...
-     Note: checking out 'f7176f4798d068d233dca5223ae4bd9c8059e830'.
+     Note: switching to 'f7176f4798d068d233dca5223ae4bd9c8059e830'.
```
Comment (amorde) [inline install_add_pod/after/execution_output.txt:33]: cc @seanreinhardtapps any idea why these changed?

Comment (seanreinhardtapps) [reply install_add_pod/after/execution_output.txt:33]: This looks to be coincidental with Git 2.27.0:
https://github.com/git/git/commit/328c6cb853d7237098569de9f94bc3d259846a08

Comment (amorde) [reply install_add_pod/after/execution_output.txt:33]: Oh this might be caused by my local git version being different than what’s on CI

Thanks!

**Comment by dnkoutso** [comment]
@amorde should we merge this? I think so right?

**Comment by amorde** [comment]
There’s a bunch of changes related to a newer git version that we shouldn’t do since that won’t show up on CI - I’ll fix it when I get the chance

**Pull request merged**